### PR TITLE
queries for nonviz metrics, pt. 1

### PR DESCRIPTION
**Pull request by JamesKunstle**
issue @ https://github.com/oss-aspen/8Knot/issues/7

Signed-off-by: James Kunstle <jkunstle@bu.edu>

### CODE REVIEW

File: metrics/avg-current-open-pr-age.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	avg(now() - pr.pr_created_at) as difference
+from
+	augur_data.pull_requests pr,
+	augur_data.repo r,
+	augur_data.repo_groups rg
+where
+	r.repo_group_id = rg.repo_group_id
+	and rg.rg_name in ('ansible')
+	and pr.repo_id = r.repo_id
+	and pr.pr_merged_at is null
```
Comment (cdolfi) [inline metrics/avg-current-open-pr-age.sql:20]: Dont need to check for both, if it is merged closed will also have a value but not the other way around

File: metrics/avg-merged-pr-age.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ * Average historical age of PR's that have been merged.
+ * 
+ * For app query, we'll be conditioning on repo_id only.
+ * Then, we can sum the total from all repos in the
+ * user's query set.
+ * 
+ * */
+
+select
+	avg(pr.pr_merged_at - pr.pr_created_at) as difference
```
Comment (cdolfi) [inline metrics/avg-merged-pr-age.sql:11]: what happened if the merged value is null?

File: metrics/avg-message-count-closed-prs.sql [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	from
+		augur_data.pull_requests pr,
+		augur_data.pull_request_message_ref prmr,
+		augur_data.repo r,
+		augur_data.repo_groups rg
+	where
+		r.repo_group_id = rg.repo_group_id
+		and rg.rg_name in ('ansible')
+		and pr.repo_id = r.repo_id
+		and prmr.pull_request_id = pr.pull_request_id
```
Comment (cdolfi) [inline metrics/avg-message-count-closed-prs.sql:30]: what limits this to only closed prs?

File: metrics/issue-tag-popularity.sql [hunk 4]
```diff
@@ -0,0 +1,23 @@
+/*
+ * Tags on issues, ordered by count.
+ * 
+ * For app query, we'll be conditioning on repo_id only.
+ * Then, we can sum the total from all repos in the
+ * user's query set.
+ * 
+ * */
+
+select
```
Comment (cdolfi) [inline metrics/issue-tag-popularity.sql:10]: im not really able to conceptualize this one. What is it?

File: metrics/avg-current-open-pr-age.sql (hunk 1 above)
Comment (JamesKunstle) [reply metrics/avg-current-open-pr-age.sql:20]: ++

File: metrics/avg-message-count-closed-prs.sql (hunk 3 above)
Comment (JamesKunstle) [reply metrics/avg-message-count-closed-prs.sql:30]: ++

File: metrics/issue-tag-popularity.sql (hunk 4 above)
Comment (JamesKunstle) [reply metrics/issue-tag-popularity.sql:10]: counts the labels of issues (bug, enhancement, etc) and ranks them by their count

File: metrics/avg-merged-pr-age.sql (hunk 2 above)
Comment (JamesKunstle) [reply metrics/avg-merged-pr-age.sql:11]: fixed

**Pull request closed without merging**